t/24-myjsonrpc.t

```diff
@@ -56,11 +59,28 @@ sub magic_close () {
     my $quit = myjsonrpc::read_json($isotovideo);
     is($quit, undef, "received magic close");
 }
+
 subtest magic_close => sub {
     my @warnings = warnings { magic_close() };
     like($warnings[0], qr{received magic close});
 };
 
+subtest 'send_json dies when buffer is empty and pipe is broken' => sub {
+    local *CORE::GLOBAL::syswrite = sub {
+        $! = 'broken pipe';
+        return 0;
+    };
+    dies_ok { send_json($child, undef) };
+};
+
+subtest 'send_json succeeds when buffer is empty and pipe is NOT broken' => sub {
```

Comment:
The successful case is already tested, so we don't need that

Reply:
Yes, I've already removed with my latest commits!